src/Illuminate/Database/Events/NoPendingMigrations.php

```diff
@@ -2,7 +2,9 @@
 
 namespace Illuminate\Database\Events;
 
-class NoPendingMigrations
+use Illuminate\Contracts\Database\Events\MigrationEvent as MigrationEventContract;
+
+class NoPendingMigrations implements MigrationEventContract
```

Comment:
For some reason this event didn't have the `MigrationEvent` contract which seemed odd to me. I can't think of a reason why it shouldn't have this.